src/attr/_make.py

```diff
@@ -229,13 +229,13 @@ class MyClassAttributes(tuple):
 
 
 def _is_class_var(annot):
-    """
-    Check whether *annot* is a typing.ClassVar.
+    """Check whether *annot* is a typing.ClassVar.
```

Comment:
This is not how we write docstrings around here! 👨‍🏫

Reply:
![](http://langa.pl/random/memes/reverse-polish-sausage.png)